src/file-input/analytics-metadata/interfaces.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export interface GeneratedAnalyticsMetadataFileInputClick {
+  action: string;
```

Comment:
I see that in most of the components, the action is a predefined string, why it's a generic string in this case?

Reply:
It is a mistake from my end, thank you for spotting